docs/general/administration/hardware-selection.md

```diff
@@ -0,0 +1,188 @@
+---
+uid: admin-hardware-selection
+title: Hardware Selection
+---
+
+# Selecting Appropriate Hardware
+
+This page covers what you need to know in order to select appropriate hardware for a Jellyfin server and take full advantage of its features (e. g. hardware acceleration).
+
+## TL;DR
+
+For a Jellyfin server, the following is recommended:
+
+### Normal Server
+
+- CPU (With dGPU): Intel Core i5-2300, AMD Ryzen 3 1200 or better. (Geekbench 6 Multicore 1500 or better)
+- CPU (Without dGPU): Intel Pentium G4560, Intel Core i3-7100 or better. (Intel 7th gen or newer Pentium or better)
+- RAM: 8GB or more
+- Storage: 32GB SSD storage for Jellyfin files and transcoding cache.
```

Comment:
We might want to bump that. 32GB for cache and file might be a bit on the lower side (e.g. trancoding 4k might already eat 20GB)

Reply:
I bumped it to 60GB.